CodeEdit/Breadcrumbs/BreadcrumbsComponent.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BreadcrumbsComponent.swift
+//  CodeEdit
+//
+//  Created by Lukas Pistrol on 18.03.22.
+//
+
+import SwiftUI
+
+struct BreadcrumbsComponent: View {
+
+	var title: String
+	var image: String
+	var color: Color
```

Comment:
are these (title,image,color) values going to change? Should these be public accessibles? 
If not I would set those as privates and change it to `let` 

Reply:
I'll change them to `let`

Reply:
and they're private now